spec/apps/partial-rerouting/partial-rerouting.spec.js

```diff
@@ -0,0 +1,129 @@
+import * as singleSpa from "single-spa";
+
+describe("partial rerouting", () => {
+  function delay() {
+    return new Promise((resolve) => setImmediate(resolve));
```

Comment:
Ah i see - you're using setImmediate to avoid having to use jest fake timers? Nice approach 👍